code/game/antagonist/outsider/storyteller.dm

```diff
@@ -0,0 +1,16 @@
+GLOBAL_DATUM(storytellers, /datum/antagonist/storyteller)
+
+/datum/antagonist/storyteller
+	id = MODE_STORYTELLER
+	role_text = "Storyteller"
+	role_text_plural = "Storytellers"
+	welcome_text = "You are a Storyteller. Basically a discount admin."
+	faction = "storyteller"
+	flags = ANTAG_NO_ROUNDSTART_SPAWN | ANTAG_OVERRIDE_JOB
+	mob_path = null
+
+	bantype = "storyteller"
+
+/datum/antagonist/storyteller/New()
+	..(1)
```

Comment:
Why are we sending a single parameter up the call chain, where `/datum/antagonist/New()` isn't taking any param? And, is it a magic number?